modules/tools/kube-prometheus-stack/values.tpl

```diff
@@ -67,6 +67,8 @@ prometheusOperator:
   configReloaderCpu: 100m
   configReloaderMemory: 25Mi
 prometheus:
+  annotations:
+    downscaler/exclude: "true"
```

Comment:
do we want to add these annotations to alertmanager / grafana too?

Reply:
We talked with Scribbles we decided we don't want grafana up after hours.

Reply:
so then grafana, alertmanager, and prometheus-node-exporter would be scaled down after hours, is that the goal?

Reply:
Yup, just having prometheus up.